mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+  return price + (price / 100) * 20;
```

Comment:
Hi  Saqib , i'm doing a code review as part of homework .
What i would suggest is in case you want a more simple  code you could use just return price + (price *0.2) .
But great job either way .

Reply:
@Bedi06 thanks for your feedback.